tools/gendocs/gendocs/microshift.go

```diff
@@ -66,11 +67,24 @@ var microshiftCommands = sets.NewString(
 	"oc registry info",
 	"oc registry login",
 	"oc replace",
+	"oc rollback",
+	"oc rollback",
```

Comment:
nit: `oc rollback` appears 2 times.

Reply:
Since CI failed I went ahead and fixed this one.